src/widget_date.lisp

```diff
@@ -0,0 +1,19 @@
+(in-package :cl-jupyter-widgets)
+
+(defclass date-picker (labeled-widget dom-widget core-widget)
+  ((value :initarg value :accessor value
+	  :type unicode
+	  :initform (unicode "")
+	  :metadata (:sync t
+			   :json-name "value"
+			   ;;what to put for **datetime_serialization?
+			  ;; https://github.com/drmeister/widget-dev/blob/master/ipywidgets6/widgets/widget_date.py#L19
```

Comment:
It turns a python datetime object to json and vice versa https://github.com/drmeister/widget-dev/blob/master/ipywidgets6/widgets/trait_types.py#L28